docs/search/criteria_reference/symbolattribute_criterion.md

```diff
@@ -0,0 +1,28 @@
+---
+description: SymbolAttribute Criterion
+---
+
+# SymbolAttributeCriterion
+
+The `SymbolAttribute` Search Criterion searches for products by [symbol attribute](symbol_attribute_type.md).
+
+## Arguments
+
+- `identifier` - identifier of the format
+- `value` - array with the values to search for
+
+## Example
+
+### PHP
+
+``` php
+<?php
```

Comment:
Please extract the code sample to a separate file in `code_samples`, as with the Luhn formula.